templates/repo/view_file.tmpl

```diff
@@ -45,7 +45,7 @@
 		</div>
 	</h4>
 	<div class="ui attached table unstackable segment">
-		<div class="file-view {{if .IsMarkup}}markdown{{else if .IsRenderedHTML}}plain-text{{else if .IsTextFile}}code-view{{end}} has-emoji">
+		<div class="file-view {{if .IsMarkup}}markdown{{if (.Repository.TocMarkdownAlways)}} auto-toc{{else if (.Repository.TocMarkdownByFlag)}} auto-toc-by-flag{{end}}{{else if .IsRenderedHTML}}plain-text{{else if .IsTextFile}}code-view{{end}} has-emoji"{{if or .Repository.TocMarkdownAlways .Repository.TocMarkdownByFlag}} data-toc="{{.i18n.Tr "repo.settings.toc.toc"}}"{{end}}>
```

Comment:
`IsMarkup` don't mean it's markdown, maybe it's org-mode .

Reply:
Didn't even know gitea support this :)
You right in `routers/repo/view.go` the `IsMarkup` flag is set if markup format via the filename.

The .org file i've tested looks quite well with toc.

Is there a problem if .org files are also rendered? May I add an additional configuration flag on which files auto-toc is rendered?

Something like
TOC_SUPPORTED_FILEEXTENSIONS (DEFAULT:  .md,.org)
so it will not interfere with external renderers. (csv could be ignored)

Reply:
There is now an configuration flag:

TOC_MARKDOWN_FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd,.org

Reply:
@Cherrg I think that's better.